src/sonic-ctrmgrd/ctrmgr/container

```diff
@@ -389,24 +403,26 @@ def main():
     args = parser.parse_args()
     kwargs = {}
 
+    ret = 0
```

Comment:
Currently unused
Do we need?

Reply:
Hi Renuka. Do you mean the variable ret? It is used below where the return value is returned at the end of function so when the process exits it will have appropriate return code.